crates/sqlite-db/src/rollover/overwrite.rs

```diff
@@ -169,7 +169,7 @@ async fn insert_revoked_commit_transaction(
     revoked: RevokedCommit,
 ) -> Result<()> {
     let revoked_tx_script_pubkey = revoked.script_pubkey.to_hex();
-    let revocation_sk_theirs = models::SecretKey::from(revoked.revocation_sk_theirs);
+    let revocation_sk_theirs = revoked.revocation_sk_theirs.map(models::SecretKey::from);
```

Comment:
❓ Is there logically a difference or is this just syntax sugar?

Reply:
Same answer as above, but for writing. So we are mapping from `secp256k1::SecretKey` to `sqlite_db::models::SecretKey`.